wiki/osu!wiki_Contribution_Guide/ph.md

```diff
@@ -0,0 +1,113 @@
+# Gabay sa Pag-ambag ng osu! wiki
+
+Maraming Salamat sa iyong interes sa paggawa ng osu! Wiki upang ito ay mapabuti! sa oras na ito, ang pagbibigay ng kontribusyon sa osu! Wiki ay nangangailangan ng mas maraming oras at bahagyang higit na pagsisikap upang maisagawa ang iyong mga pagbabago. Ang mabilisang gabay na ito ay inilaan para sa mga nagsisimula pa lamang at hindi pa gumagawa sa Github at ang osu!wiki workflow sa bago. Kung mayroon ka nang karanasan sa pag gamit ng GitHub, huwag mag-atubiling laktawan ang gabay na ito at sundin ang pangkaraniwang work-branch workflow para sa mga update na nilalaman ng imbakan na ito.

```

Comment:
`osu!wiki` -> `osu! wiki`
Can you do that for every instance? Including the name of the folder to `osu!_wiki_Contribution_Guide`. A recent PR was pushed with these minor changes.

Reply:
Sure